app/bundles/LeadBundle/EventListener/OwnerSubscriber.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\LeadBundle\EventListener;
+
+use Mautic\EmailBundle\EmailEvents;
+use Mautic\EmailBundle\Event\EmailBuilderEvent;
+use Mautic\EmailBundle\Event\EmailSendEvent;
+use Mautic\LeadBundle\Model\LeadModel;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Class EmailSubscriber.
```

Comment:
Remove this comment as it's false and even if correct it does not add any value

Reply:
Fixed.

Reply:
*Note* The code base is littered throughout with these. Maybe a php-cs-fixer could be added to remove these blocks of code where the pattern matches that it adds no value.